docs/installation.md

```diff
@@ -29,32 +29,71 @@ You are now ready to clone and install TopoStats.
 
-## Cloning and Installing
+## Installing TopoStats
 
-Currently TopoStats has not been released to the [PyPi](https://pypi.org) repository (although a release is planned). In
-the mean time you have to clone the repository from GitHub and install it. If you do not have Git already installed
-please see [Git](#git). Work is on-going on the `dev` branch and so
-to test that you need to "checkout" this branch prior to installing it.
+There are two options for installing TopoStats depending on your usage
+
+1. [**Python Package Index**](https://pypi.org/) - appropriate if you are just using TopoStats and don't need to dig into
+   the code.
+2. Cloning the GitHub Repository - if you want to look at the code, contribute to it, debug errors or perhaps test a new
+   feature before a release.
+
+### PyPI Installation
+
+After activating your `topostats` Conda environment you can install TopoStats from PyPI using the following command.
 
 ``` bash
-git clone -b dev https://github.com/AFM-SPM/TopoStats.git
-```
-``` bash
-cd TopoStats
+pip install topostats
 ```
+
+This will install TopoStats under your virtual environment and the command `run_topostats` will be available at the
+command line. You will probably want to download a copy of the
+[`default_config.yaml`](https://raw.githubusercontent.com/AFM-SPM/TopoStats/main/topostats/default_config.yaml) to edit
+and use for running your analysis. Please see the [usage](usage) section for more information on running TopoStats.
+
+### Cloning from GitHub
+
+If you do not have Git already installed please see [Git](#git). If you intend to contribute to the development of
+TopoStats please read through the [contributing](contributing) section.
+
+If you are familiar with the command line then you can
+clone and install TopoStats with the following _after_ activating your virtual environment.
+
 ``` bash
-git checkout dev
+git clone https://github.com/AFM-SPM/TopoStats.git
+# If you have SSH access configured to GitHub then you can use
+git clone git@github.com:AFM-SPM/TopoStats.git
 ```
+
+If you wish to make changes to the code then install in editable mode, i.e. `pip install -e .`.
+
+
+#### Cloning Using GitKraken
+
+If you are using GitKraken you can clone the repository by selecting "Clone" and then "GitHub.com" and typing
+`TopoStats` into the box next to "Repository to Clone" and you should be presented with the option of selecting
+"TopoStats" from the AFM-SPM organisation.
+
+Alternatively you can "Clone with URL" and enter `https://github.com/AFM-SPM/TopoStats.git` as the URL to clone from,
+selecting a destination to clone to.
+
+
+#### Installing TopoStats from the Cloned Repository
+
+Once cloned you will have to open a Terminal and navigate to the directory you cloned and _after_ activating your
+virtual environment install TopoStats with the following.
+
 ``` bash
+cd /path/to/where/topostats/was/cloned/TopoStats
```

Comment:
I have found it very hard to indicate to users where to insert things and not just copy / paste the code line. I know of no perfect way to do it and have no solid suggestions here, only that perhaps users will not be able to tell that they should keep the `/TopoStats` at the end?

Reply:
A perennial challenge, can't think of a way around it either.